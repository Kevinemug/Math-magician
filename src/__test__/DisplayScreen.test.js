import React from 'react';
import renderer from 'react-test-renderer';
import DisplayScreen from '../components/displayScreen'

it('renders DisplayScreen correctly', () => {
  const tree = renderer.create(<DisplayScreen />)
  expect(tree).toMatchSnapshot()
});