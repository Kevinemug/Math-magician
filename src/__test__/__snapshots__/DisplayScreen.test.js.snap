// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DisplayScreen correctly 1`] = `
<div
  className="displayscreen-cont"
>
  <span
    className="display-screen"
  />
</div>
`;
